### PR TITLE
doc: rename example directories

### PR DESCRIPTION
**Pull request by shiipou**
Better explanation of Example directory contents

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -9,22 +9,27 @@ Here you will find some example repositories linked by dependencies defined in a
...
+protocol-lib: string-lib
+core-code: string-lib, protocol-lib
+desktop-app: core-code
+mobile-app: core-code
+web-app: core-code
 ```
 
-As you can see, we have 3 Git repositories that depend on each other. In our example, repository "my-core-code" needs repository "my-library" to compile. And as it is quite a pain to always have to manually edit the right dependency files, this action will take care of it if it is well configured.
-If we continue our exploration, the "my-dev-app" repository also requires the "my-library" repository in addition to the "my-core-code" repository. Projects of a certain size can have pretty complex mesh links which can be a source of errors.
+As you can see, we have a total of 6 Git repositories that depend on each other. In our example, repository "core-code" that will contain main code of the app that will be shared on each platform (desktop-app, mobile-app, and web-app). That core-code needs repository "string-lib" and "protocol-lib" to compile. And as it is quite a pain to always have to manually edit the right dependency files, this action will take care of it if it's well configured.
```
Comment (jonas-martinez) [inline examples/README.md:19]:
```suggestion
As you can see, we have a total of 6 Git repositories that depend on each other. In our example, repository "core-code" that will contain the main code of the app that will be shared on each platform (desktop-app, mobile-app, and web-app). That core-code needs the repositories "string-lib" and "protocol-lib" to compile. And as it is quite a pain to always have to manually edit the right dependency files, this action will take care of it if it's well configured.
```

File: examples/README.md [hunk 2]
```diff
@@ -9,22 +9,27 @@ Here you will find some example repositories linked by dependencies defined in a
...
+If we continue our exploration, the "protocol-lib" repository also requires the "string-lib" repository.
+Projects of a certain size can have pretty complex mesh links which can be a source of errors.
 
 ### 2. Understanding links between each workflows
 
-When you publish a new release on the repository "my-library", this action will automatically create a Pull Request on the repository "my-core-code" and "my-dev-app" to update the correct files and change it's own link in dependancy file(s).
-And now, when you merge the PR on the repository "my-core-code", and create a new release, it will update the PR on the repository "my-dev-app" to add this changes.
+When you publish a new release on the repository "string-lib", this action will automatically create a Pull Request on the repository "protocol-lib" and "core-code" to update the correct files and change it's own link in dependancy file(s).
```
Comment (jonas-martinez) [inline examples/README.md:25]:
```suggestion
When you publish a new release on the repository "string-lib", this action will automatically create a Pull Request on the repository "protocol-lib" and "core-code" to update the correct files and change its own link in dependency file(s).
```

File: examples/README.md [hunk 3]
```diff
@@ -9,22 +9,27 @@ Here you will find some example repositories linked by dependencies defined in a
...
+And now, when you merge the PR on the repository "protocol-lib", and create a new release, it will update the previously created PR on the repository "core-code" to add this changes if it's not already merged.
 
-Now we can accept changes on the repository "my-dev-app" to update it's version of "my-library" and "my-core-code".
+Now we can accept changes on the repository "core-code" to update it's version of "string-lib" and "protocol-lib".
```
Comment (jonas-martinez) [inline examples/README.md:28]:
```suggestion
Now we can accept changes on the repository "core-code" to update its version of "string-lib" and "protocol-lib".
```

File: examples/README.md [hunk 4]
```diff
@@ -9,22 +9,27 @@ Here you will find some example repositories linked by dependencies defined in a
...
+This will batch upgrade at once the version in "desktop-app", "mobile-app", and "web-app".
 
 ### 3. Go deeper for an even better workflow
 
-This is be awesome to use in your projects with sementic-release to make releases even faster and erase error of dependency managments
+This is be awesome to use in your projects with "sementic-release" to make releases even faster and erase error of dependency managments
```
Comment (jonas-martinez) [inline examples/README.md:33]:
```suggestion
This is awesome to use in your projects with "semantic-release" to make releases even faster and avoid errors of dependency management.
```

File: examples/web-app/.github/workflows/create_or_update_pr.yml [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    timeout-minutes: 10
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup yq
+        uses: shiipou/setup-yq-action@v2.0.2
+      - name: create_pr
+        uses: lenra-io/create-or-update-pr-action@v1.0.0-beta.2
+        with:
+          name: 'Update dependecies'
```
Comment (jonas-martinez) [inline examples/web-app/.github/workflows/create_or_update_pr.yml:29]:
```suggestion
          name: 'Update dependencies'
```

File: README.md [hunk 6]
```diff
@@ -12,7 +12,7 @@ To learn accurately how to use it, you must see the [guide here](/examples)
 ```yaml
       # Call `Create or update PR` GitHub action
       - name: create_pr
-        uses: lenra-io/create-or-update-pr-action@v1
+        uses: lenra-io/create-or-update-pr-action@main
         with:
           # The name of the PR to be created.
           name: 'Update dependecies'
```
Comment (jonas-martinez) [inline README.md:18]:
```suggestion
          name: 'Update dependencies'
```

File: examples/core-code/.github/workflows/create_or_update_pr.yml [hunk 7]
```diff
@@ -22,19 +22,18 @@ jobs:
...
         uses: actions/checkout@v2
       - name: Setup yq
-        id: setup-yq
-        uses: shiipou/setup-yq-action@v2.0.1
+        uses: shiipou/setup-yq-action@v2.0.2
       - name: create_pr
-        uses: lenra-io/create-or-update-pr-action@v1
+        uses: lenra-io/create-or-update-pr-action@v1.0.0-beta.2
         with:
           name: 'Update dependecies'
```
Comment (jonas-martinez) [inline examples/core-code/.github/workflows/create_or_update_pr.yml:29]:
```suggestion
          name: 'Update dependencies'
```

File: examples/desktop-app/.github/workflows/create_or_update_pr.yml [hunk 8]
```diff
@@ -22,19 +22,18 @@ jobs:
...
         uses: actions/checkout@v2
       - name: Setup yq
-        id: setup-yq
-        uses: shiipou/setup-yq-action@v2.0.1
+        uses: shiipou/setup-yq-action@v2.0.2
       - name: create_pr
-        uses: lenra-io/create-or-update-pr-action@v1
+        uses: lenra-io/create-or-update-pr-action@v1.0.0-beta.2
         with:
           name: 'Update dependecies'
```
Comment (jonas-martinez) [inline examples/desktop-app/.github/workflows/create_or_update_pr.yml:29]:
```suggestion
          name: 'Update dependencies'
```

File: examples/mobile-app/.github/workflows/create_or_update_pr.yml [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+    timeout-minutes: 10
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup yq
+        uses: shiipou/setup-yq-action@v2.0.2
+      - name: create_pr
+        uses: lenra-io/create-or-update-pr-action@v1.0.0-beta.2
+        with:
+          name: 'Update dependecies'
```
Comment (jonas-martinez) [inline examples/mobile-app/.github/workflows/create_or_update_pr.yml:29]:
```suggestion
          name: 'Update dependencies'
```

File: examples/protocol-lib/.github/workflows/create_or_update_pr.yml [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+    timeout-minutes: 10
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup yq
+        uses: shiipou/setup-yq-action@v2.0.2
+      - name: create_pr
+        uses: lenra-io/create-or-update-pr-action@v1.0.0-beta.2
+        with:
+          name: 'Update dependecies'
```
Comment (jonas-martinez) [inline examples/protocol-lib/.github/workflows/create_or_update_pr.yml:29]:
```suggestion
          name: 'Update dependencies'
```

**Review by jonas-martinez — CHANGES_REQUESTED** [review]

**Comment by shiipou** [comment]
@jonas-martinez Thanks for the review, I've applied your suggestions.

**Review by jonas-martinez — APPROVED** [review]
Looks good to me :)

**Pull request merged**